organising/organising-a-meeting.md

```diff
@@ -0,0 +1,164 @@
+
+### What if it doesn't go smoothly
+
+Hopefully all that is needed is for you to pick the top 2 or 3 volunteers who are already earmarked for the meeting you are organising and email them to confirm the date and their timeslot.  They get back to you and the meting is full of speakers.
+
+It doesn't always go that smoothly.
+
+* Speakers pull out
+* Speakers just don't respond
+* We don't have anyone earmarked for that meeting
+
+In this case you'll need to actually find some speakers as well as confirm them.  Check out [our tips on populating the speaker backlog](/organising/maintaining-the-community#populate-the-speaker-backlog) if you need to find speakers.  Adding an urgent "_for this month's meeting_" plea can help.
+
+## Announce the meeting
+
+Once you have at least one speaker confirmed you are ready to announce the meeting.  This means:
+
+1. [Set up the eventbrite page so people get reserve their place](/organising/organising-a-meeting#set-up-the-eventbrite-page)
+2. [Schedule the zoom meeting](/organising/organising-a-meeting#schedule-the-zoom-meeting)
+3. [Write up the meeting on lrug.org](/organising/organising-a-meeting#write-up-on-lrugorg)
+4. [Tell the mailing list about it](/organising/organising-a-meeting#tell-the-mailing-list-about-the-meeting)
+5. [Tweet about it](/organising/organising-a-meeting#tweet-about-the-meeting)
+
+Ideally we do all this the week after the previous meeting, but sometimes we do it really close to the event if we're struggling to get speakers.  It can be useful to write up placeholder versions and publish them, so that people will at least get the event into their calendars.
+
+If you have more than one speaker it can be useful to trickle the speaker announcements out over a couple of weeks in order to keep reminding people.
+
+### Set up the eventbrite page
+
+We use eventbrite to collect attendee details and manage how many people can come.  Our zoom account has 100 places so we limit the eventbrite page to 95 tickets to leave room for the host and speakers.
+
+We like to be creative with this page, add a fun header image and create amusing ticket types.  We tend not to put full meeting details into this and just point people at lrug.org where the write up is.
+
+* set ticket sales to close at 5:30pm on the day
+* limit total ticket sales to 95
+* add a link to the zoom meeting as a plain link to the "online event page" - we haven't linked our zoom account, and the link works just fine.
+* check that there are scheduled emails to attendees on the day to send the link out to them before 6:30pm
+* enable the waiting list in the off chance the meeting "sells out"
+
+### Schedule the zoom meeting
+
+We schedule the zoom meeting in advance so we can:
+
+1. add the link to the "online event page" in eventbrite
+2. pass on the link to the speakers well in advance
+3. get the recording properly named for the meeting
```

Comment:
Maybe move this section above the Eventbrite one, so that the zoom meeting link is already available for use there?